app.py

```diff
@@ -12,18 +21,22 @@ def execute_board(site,  username, repo):
     '''Take Arguments from command line OR Read configuration files, which is YAML format'''
     #print (site)
     #print (username) 
-    if site == 'config'  and username == 'config' and repo == 'config':
+    if username == 'config' and repo == 'config':
           print ("The Application will read username and code sites from configuraiton file")
           with open('config.yml', 'r') as f:
```

Comment:
First, check that the file is present or not, you can do that using `os` module.

Reply:
Right now this PR does not focus on config file parsing, we will check this in #23